test/unit/points.js

```diff
@@ -8,15 +8,15 @@ import { getCollide } from '../../src/basic/points';
 
 describe('Points', () => {
   describe('.getCollide', () => {
-    xit('Should return false when marker-allow-overlap is not defined', () => {
+    it('Should return false when marker-allow-overlap is not defined', () => {
       const c3ss = Utils.getShader('#layer {  marker-line-color: #FFF; }');
       expect(getCollide(c3ss)).to.equal(false);
```

Comment:
The reason for the `xit` is that this is returning "false" (string) instead `false`(boolean)